mos_tests/neutron/python_tests/base.py

```diff
@@ -141,3 +178,29 @@ def check_vm_is_connectable(self, vm):
             wait(lambda: remote.execute(cmd)['exit_code'] == 0,
                  interval=10, timeout=3 * 10,
                  timeout_msg=error_msg)
+
+    def check_vm_is_accessible_with_ssh(self, vm_ip, pkeys=None):
+        """Check that instance is accessible with ssh via floating_ip.
+
+        :param vm_ip: floating_ip of instance
+        :param pkeys: ip of instance to ping
+        """
+        error_msg = 'Instance with ip {0} is not accessible with ssh.'\
+            .format(vm_ip)
+
+        def is_accessible():
+            try:
+                with self.env.get_ssh_to_cirros(vm_ip, pkeys) as vm_remote:
+                    vm_remote.execute("date")
+                    return True
+            except Exception:
+                return False
+
+        wait(lambda: is_accessible(),
```

Comment:
I've replaced devops wait with waiting wait (https://pypi.python.org/pypi/waiting). So this won't work after merge. You should rename args of wait call